### PR TITLE
General fixes

### PR DESCRIPTION
**Pull request by pratishshr**
- Fixes
- Fetch last status of change log

### CODE REVIEW

File: src/models/StatusChange.js [hunk 1]
```diff
@@ -14,6 +14,22 @@ class StatusChange extends db.Model {
   }
 
   /**
+   * Fetch a statusChange by its name.
+   *
+   * @param  {String} id
+   * @return {Promise}
+   */
+  static async fetchByName(name) {
+    let statusChange = await new StatusChange({ name }).orderBy('created_at', 'DESC').fetch();
```
Comment (romitamgai) [inline src/models/StatusChange.js]: try catch block here?

Comment (romitamgai) [inline src/models/StatusChange.js]: nevermind handled in persistence.js

Comment (pratishshr) [inline src/models/StatusChange.js]: was done earlier, later handled in service layer

File: src/models/StatusLog.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  }
+
+  /**
+    * Create a new statusLog.
+    *
+    * @param  {Object} statusLog
+    * @return {Promise}
+    */
+  static async create({ name, status }) {
+    let statusLog = await new StatusLog({ name, status }).save();
```
Comment (kabirbaidhya) [inline src/models/StatusLog.js]: I don't think we need `async` and `await` here. `save()` would directly return `Promise`.
Just `return new StatusLog({ name, status }).save();` should do it.

**Pull request merged**